pkg/actions/cluster/unowned_test.go

```diff
@@ -152,6 +163,180 @@ var _ = Describe("Delete", func() {
 			Expect(fakeStackManager.DeleteStackBySpecCallCount()).To(Equal(1))
 			Expect(*fakeStackManager.DeleteStackBySpecArgsForCall(0).StackName).To(Equal("fargate-role"))
 		})
+
+		It("ignoring nodes draining error when force flag used", func() {
```

Comment:
I think some more context to each test will make it slightly easier to read:

```suggestion
            When("force is true", func(){
		It("ignores errors when draining the nodes", func() {
```

I think this goes for the rest of the new tests too, its often easier to see the `When` describe whats changed, and the `It` the desired behaviour :smile:

Reply:
Added these contexts for the unit tests